cluster/addons/metadata-agent/stackdriver/metadata-agent-rbac.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:metadata-agent
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+rules:
+- apiGroups:
+  - ""
+  resources:
```

Comment:
The agent will also watch the cluster and services resources, as well as workloads in the future. Does it make sense to add those permissions now, or wait until they're actually needed?

Reply:
Added services. I think that we actually don't have cluster resource right now (it's certainly not a part of core api).

We should add all permissions now. If Metadata Agent starts watching some resource without permissions, the release will be blocked on Kubernetes release, which we want to avoid.

Reply:
Fine, can we then also add deployments, daemon sets, replica sets, replication controllers, stateful sets, and other workloads?